packages/core/src/domain/configuration.ts

```diff
@@ -256,6 +256,11 @@ function getHost(intakeType: IntakeType, endpointType: EndpointType, site: strin
   return `${endpoint}.browser-intake-${suffix}`
 }
 
+function getIntakeType(site: string, userConfiguration: UserConfiguration) {
+  // TODO when new intake will be available for gov, only allow classic intake for us and eu
```

Comment:
Couldn't we do this already? something like:
```ts
if (!userConfiguration.useAlternateIntakeDomains && (site === 'datadoghq.com' || site === 'datadoghq.eu')) {
  return 'classic'
}
return 'alternate'
```

Reply:
this would not allow classic intake for gov or am I missing something?

Reply:
I'm not sure about the specific rules we should apply, but it seems that new datacenters won't have classic intake anymore. Thus, I think that it would be better if we had a list of sites that *support* classic intakes (since it is not likely to change in the future), instead of a list of sites that *don't support* classic intakes (since this list is likely to grow when new datacenters are added).

Reply:
It is what I have in mind too but I'd prefer to not break rum on dd gov app with this PR.
We could implement this logic now and define gov as supporting classic but we would still need an extra PR to remove it. 
no strong opinion about one way or the other.

> It seems that our intake host logic gets more complex, and I wonder if we shouldn't centralize/colocate this logic.

agree, we could wait for the PR for gov since this part should not change too much in the mean time and it would avoid to deal with conflicts for now.

Reply:
I'm fine either way. We should not take conflicts into account: my PR doesn't change the same part of `configuration.ts` so we shouldn't have any conflict. We can think again how to improve on this once everything gets merged.